packages/react/src/components/Slug/SlugExamples.mdx

```diff
@@ -0,0 +1,97 @@
+# Slug
+
+[Source code](https://github.com/carbon-design-system/carbon/tree/main/packages/react/src/components/Slug)
+|
+[Slug Release Status](https://airtable.com/appCAqlGVN8tRUbAp/shr71ZyLlIGORz3Vh/tblHqPusgkK8hIeHo)
+|
+[Using AI-enhanced components in V10](https://github.com/carbon-design-system/carbon-for-ai/blob/main/docs/support-for-carbon-v10.md)
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Slug anatomy](#slug-anatomy)
+- List of components that accept `Slug`
+  - [Checkbox](https://react.carbondesignsystem.com/?path=/story/experimental-unstable-slug-examples--checkbox)
```

Comment:
Not a big deal, but I noticed these open in new browser tabs. When we cross-link stories, do we typically use relative paths or absolute like this?